include/mme-app/actionHandlers/actionHandlers.h

```diff
@@ -358,16 +388,31 @@ namespace mme
         ***********************************************/
         static SM::ActStatus process_ho_failure(SM::ControlBlock& cb);                
 
+        /**********************************************
+        * Action handler : process_ho_fwd_acc_cntx_noti
+        ***********************************************/
+        static SM::ActStatus process_ho_fwd_acc_cntx_noti(SM::ControlBlock& cb);                
+
         /**********************************************
         * Action handler : process_ho_notify
         ***********************************************/
         static SM::ActStatus process_ho_notify(SM::ControlBlock& cb);                
 
+        /**********************************************
+        * Action handler : process_ho_req_ack
+        ***********************************************/
+        static SM::ActStatus process_ho_req_ack(SM::ControlBlock& cb);                
```

Comment:
What type of HO is this ?

Reply:
This is for inter-mme HO type, but at the target MME side.